presto-main/src/test/resources/io/prestosql/execution/scheduler/topology-new.txt

```diff
@@ -0,0 +1,3 @@
+192.168.0.1 /region1/rack1/machine5
```

Comment:
I think we need a line with multiple spaces between host and location, and a line with leading spaces.